Training6/training-6-sagas-login/src/reducers/LoginReducer.js

```diff
@@ -0,0 +1,41 @@
+import { types } from "../actions/index.js";
+
+const initialState = {
+  requesting: false,
```

Comment:
I see you save successfully in the state but not see the token, we should save it in store